docker-hub/builds/index.md

```diff
@@ -299,6 +305,10 @@ You can enable the BuildKit builder by setting the `DOCKER_BUILDKIT=1`
 Refer to the [build images with BuildKit](../../develop/develop-images/build_enhancements.md)
 page for more information on BuildKit.
 
+> **Note**
+>
+> Autobuilds use the buildKit build-system by default. If you want to use the legacy Docker build-system, add the [environment variable](index.md#environment-variables-for-builds){: target="_blank" rel="noopener" class="_"} Docker_BUILDKIT=0.
```

Comment:
Nit: I suggest to use only uppercase letters for environment variables: `DOCKER_BUILDKIT=0`